forge/comms/index.js

```diff
@@ -43,7 +44,7 @@ module.exports = fp(async function (app, _opts, next) {
 
         app.ready().then(async () => {
             // Once the whole platform is ready, tell the client to connect
-            await client.init()
+            return await client.init()
```

Comment:
This is a confusing mix of promises and async behaviour. I chose not to refactor it for risk of including a regression.